.eslintrc

```diff
@@ -1,7 +1,8 @@
 {
   "parser": "babel-eslint",
   'env': {
-    'es6': true
+    'es6': true,
+    'node': true
```

Comment:
The issue is that there are a few other folders (config, test, etc.) which are node-only. I think it's better to leave `'node': true` as the default, and explicitly override it as necessary. I just pushed a commit to override it in the `client/` folder.